testsuite/manualmode/src/test/resources/org/jboss/as/test/manualmode/management/persistence/test.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" ?>
+
+<server xmlns="urn:jboss:domain:19.0">
```

Comment:
Domain mode was not supported according to https://github.com/wildfly/wildfly-proposals/blob/be6bfece2ab58e46ea2f26c5f30e211a0f4c4e56/core/WFCORE-5343_Support_YAML_files_for_configuration_customization.adoc#non-requirements

Why are you testing the feature with a XML domain configuration?